src/Interpreters/Context.h

```diff
@@ -395,7 +395,7 @@ class Context: public std::enable_shared_from_this<Context>
     /// proton: starts. Parameters for time predicates of main table
     TimeParam time_param;
 
-    using DataStreamSemanticCache = std::unordered_map<std::string, Streaming::DataStreamSemanticEx>;
+    using DataStreamSemanticCache = std::unordered_map<std::string, std::pair<Streaming::DataStreamSemanticEx, bool>>;
```

Comment:
Why do not put this bool variable into 'DataStreamSemanticEx'?

Reply:
Yes, it's a good way